source/extensions/filters/common/lua/wrappers.cc

```diff
@@ -121,7 +121,7 @@ void MetadataMapHelper::createTable(lua_State* state,
   lua_createtable(state, 0, fields.size());
   for (const auto& field : fields) {
     int top = lua_gettop(state);
-    lua_pushstring(state, field.first.c_str());
+    lua_pushlstring(state, field.first.c_str(), field.first.length());
```

Comment:
stick with .size() or .length() for consistency, though they're synonyms. I'm used to .size(), but don't know if envoy uses one or the other more frequently. Basic github search has .size() (~600 matches) win out vs .length() (~100 matches).

Reply:
Changed it to `.size()`. Thanks!